Octokit/Models/Request/IssueRequest.cs

```diff
@@ -140,6 +140,27 @@ public enum ItemState
         /// </summary>
```

Comment:
Should probably add a XmlDoc comment on this guy stating "Items that are open OR closed"